pgcat.toml

```diff
@@ -0,0 +1,21 @@
+# This is an example of the most basic config
+# that will mimic what PgBouncer does in transaction mode with one server.
+
+[general]
+host = "0.0.0.0"
+port = 6433
+admin_username = "sentry"
+admin_password = "sentry"
+
+[pools.sentry.users.0]
+username = "sentry"
+password = "sentry"
+pool_size = 100
+min_pool_size = 100
```

Comment:
Should we provide in-code docs here to describe what's the difference between this `pool_size` compared to postgres' pool size?